deployments/helm/nvidia-device-plugin/templates/daemonset-mps-control-daemon.yml

```diff
@@ -61,6 +61,7 @@ spec:
         command: [mps-control-daemon, mount-shm]
         securityContext:
           privileged: true
+          {{- include "mps-control-daemon.securityContext" . | nindent 10 }}
```

Comment:
Note that this helper template is not defined.

Reply:
Removed this as well